### PR TITLE
[FEATURE] Add MigrateAddPageTSConfigToPageTsConfigFileRector

### PR DESCRIPTION
**Pull request by sabbelasichon**
Resolves: #3701

### CODE REVIEW

File: tests/Rector/v13/v0/MigrateAddPageTSConfigToPageTsConfigFileRector/Fixture/extension3/ext_localconf.php.inc [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Ssch\TYPO3Rector\Tests\Rector\v13\v0\MigrateAddPageTSConfigToPageTsConfigFileRector\Fixture;
+
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+
+$variable = 'ext_key';
+
+\TYPO3\CMS\Core\Utility\ExtensionManagementUtility::addPageTSConfig(
+    '@import "EXT:' . $variable . '/Configuration/TSconfig/*/*.tsconfig"'
```
Comment (simonschaufi) [inline tests/Rector/v13/v0/MigrateAddPageTSConfigToPageTsConfigFileRector/Fixture/extension3/ext_localconf.php.inc:10]: could we do something about this line because i see many people do it this way and then the code won't be migrated which sucks.

Comment (sabbelasichon) [reply tests/Rector/v13/v0/MigrateAddPageTSConfigToPageTsConfigFileRector/Fixture/extension3/ext_localconf.php.inc:10]: I have tried it with a naive approach. I guess there are edge case where people doing crazy things. But let´s give it a try and see what will be reported.

File: tests/Rector/v13/v0/MigrateAddPageTSConfigToPageTsConfigFileRector/MigrateAddPageTSConfigToPageTsConfigFileRectorTest.php [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    public static function provideData(): Iterator
+    {
+        yield 'Test that new page.tsconfig is created with correct content' => ['extension1'];
+
+        yield 'Test that content is appended to existing page.tsconfig file' => [
+            'extension2',
+            '# page.tsconfig file exist',
+        ];
+
+        yield 'est that new page.tsconfig is created with correct content but unresolvable content cannot be migrated properly' => [
```
Comment (simonschaufi) [inline tests/Rector/v13/v0/MigrateAddPageTSConfigToPageTsConfigFileRector/MigrateAddPageTSConfigToPageTsConfigFileRectorTest.php:68]: Test

**Review by simonschaufi — APPROVED** [review]
